### PR TITLE
Openshift haproxy promcat

### PR DESCRIPTION
**Pull request by Mak0tin**

### CODE REVIEW

File: resources/openshift-haproxy-router/alerts.yaml [hunk 1]
```diff
@@ -1,79 +1,78 @@
...
-        expr: |
-          sum by (route)(rate(haproxy_server_connection_errors_total{route!=""}[5m])) > 0
-        for: 10m
-        labels:
-          severity: page
-        annotations:
-          summary: Recurring connection errors in route {{$labels.route}}
+        - alert: '[OpenShift-HAProxy-Router] Router Down'
+          expr: |
+            absent((count(haproxy_process_start_time_seconds) < 1))
```
Comment (carlosadiegosysdig) [inline resources/openshift-haproxy-router/alerts.yaml:17]: I think the 'absent' alerts must end with a '== 1'

File: resources/openshift-haproxy-router/dashboards.yaml [hunk 2]
```diff
@@ -29,10 +29,10 @@ configurations:
     * Frontend Connections
     * Frontend Bytes Out
     * Frontend HTTP Requests
-  file: include/HAProxy_OC_Ingress_Overview-sysdig-dashboard.json
+  file: include/HAProxy_OC_Ingress_Overview-sysdig-dashboard-v4.7.json
 - name: 'HAProxy OC Service Golden Signals'
   kind: Sysdig
-  image: 'openshift-haproxy-router/images/HAProxy_OC_Service_Golden_Signals-sysdig-dashboard.png'
```
Comment (carlosadiegosysdig) [inline resources/openshift-haproxy-router/dashboards.yaml:35]: Should we also delete the image from the 'images' folder?

Comment (Mak0tin) [reply resources/openshift-haproxy-router/dashboards.yaml:35]: Yes, deleted

File: resources/openshift-haproxy-router/alerts.yaml (hunk 1 above)
Comment (Mak0tin) [reply resources/openshift-haproxy-router/alerts.yaml]: Fixed

File: resources/openshift-haproxy-router/alerts.yaml [hunk 3]
```diff
@@ -1,79 +1,78 @@
...
+          expr: |
+            sum by (route)(haproxy_server_current_queue) > 0
+          for: 10m
+          labels:
+            severity: critical
+          annotations:
+            description: This alert triggers when a queue is not empty in a route for 10m.
+        - alert: '[OpenShift-HAProxy-Router] High error rate in route'
+          expr: |
+            sum by (route) (rate(haproxy_server_http_responses_total{code!="2xx"}[5m])) /sum by (route) (rate(haproxy_server_http_responses_total[5m]))
```
[the remark this answers is not in the document]
Comment (Mak0tin) [reply resources/openshift-haproxy-router/alerts.yaml]: Fixed

**Pull request merged**